entrypoint.sh

```diff
@@ -17,14 +17,18 @@ if [[ -z "$GIT_EMAIL" ]]; then
   exit 1
 fi
 
+if [[ -z "$BUNDLER_VERSION" ]]; then
```

Comment:
https://help.github.com/ja/actions/automating-your-workflow-with-github-actions/metadata-syntax-for-github-actions#

please change.

1. set this variable as inputs in action.yml
2. use `$INPUT_BUNDLER_VERSION`